models/post.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+//const schema =mongoose.schema;
+
+const schema = new Schema({
+    title: String,
+    contents: String,
+    summary: String
+});
```

Comment:
While title, contents and summary are all good properties for a blog post, they don't really apply for our model `Topic`. 

I'll leave it to you to decide how the model for `Topic` should look (hint: have a look at the table on the main homepage). Let's discuss further once you submit changes for this PR. Thanks!